### PR TITLE
Add IntParticles (int32)

### PR DESCRIPTION
**Pull request by joeljonsson**
- add IntParticles to allow additional functionality in the napari viewer, e.g. converting image layers to labels (Note: IntParticles are not supported in most pyapr functionality)
- add a brief quick start guide with pictures and code snippets to the readme

### CODE REVIEW

File: .github/workflows/build-deploy.yml [hunk 1]
```diff
@@ -45,21 +45,21 @@ jobs:
       # Indicates the location of the vcpkg as a Git submodule of the project repository.
       VCPKG_ROOT: ${{ github.workspace }}/external/LibAPR/vcpkg
       CIBW_ENVIRONMENT_WINDOWS: EXTRA_CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=D:\\a\\pyapr\\pyapr\\external\\LibAPR\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake -DVCPKG_MANIFEST_DIR=D:\\a\\pyapr\\pyapr\\external\\LibAPR\\"
-      CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-*"
+      CIBW_BUILD: "cp38-* cp39-* cp310-* cp311-*"
       CIBW_SKIP: "*musllinux*"
       CIBW_ARCHS: "auto64"
-      CIBW_MANYLINUX_X86_64_IMAGE: "manylinux_2_24"
+      #CIBW_MANYLINUX_X86_64_IMAGE: "manylinux_2_28"
```
Comment (cheesema) [inline .github/workflows/build-deploy.yml:51]: Why not just delete this?

File: .github/workflows/build-deploy.yml [hunk 2]
```diff
@@ -45,21 +45,21 @@ jobs:
...
       CIBW_BUILD_VERBOSITY: 1
-      CIBW_REPAIR_WHEEL_COMMAND_MACOS: "pip uninstall -y delocate && pip install git+https://github.com/Chia-Network/delocate.git && delocate-listdeps {wheel} && delocate-wheel -w {dest_dir} -v {wheel}"
+      CIBW_REPAIR_WHEEL_COMMAND_MACOS: "pip uninstall -y delocate && pip install git+https://github.com/matthew-brett/delocate.git && delocate-listdeps {wheel} && delocate-wheel -w {dest_dir} -v {wheel}"
```
Comment (cheesema) [inline .github/workflows/build-deploy.yml:53]: why not just use the delocate on pypi?

**Review by cheesema — CHANGES_REQUESTED** [review]
@joeljonsson looks good, what brought about the addition on the Int, I just had some questoin regarding the CI configs.

**Comment by joeljonsson** [comment]
> @joeljonsson looks good, what brought about the addition on the Int, I just had some questoin regarding the CI configs.

It's basically just to allow to convert between `image` and `labels` layer in napari. It (and many other types) could be added to the processing, but with the way we handle things at the moment its just kind of a hassle

File: .github/workflows/build-deploy.yml (hunk 2 above)
Comment (joeljonsson) [reply .github/workflows/build-deploy.yml:53]: Good point, I can try it. I believe you originally added this line, with an url to a fork which seemed to now be out of date

File: .github/workflows/build-deploy.yml (hunk 1 above)
Comment (joeljonsson) [reply .github/workflows/build-deploy.yml:51]: will do

**Pull request merged**